src/ILCompiler.Compiler/src/IL/Stubs/StartupCode/StartupCodeMainMethod.cs

```diff
@@ -58,7 +57,7 @@ public override MethodIL EmitIL()
             ILEmitter emitter = new ILEmitter();
             ILCodeStream codeStream = emitter.NewCodeStream();
 
-            ModuleDesc developerExperience = _typeSystemContext.GetModuleForSimpleName("System.Private.DeveloperExperience.Console", false);
+            ModuleDesc developerExperience = Context.ResolveAssembly(new AssemblyName("System.Private.DeveloperExperience.Console"), false);
```

Comment:
Thanks for cleaning this up ... I forgot that there is method to load by assembly name on the base type system context.